### ISSUE TITLE
"Unknown word" errors

### ISSUE THREAD

**Issue by onlywei** [AtomLinter/linter-stylelint#52]
While using this plugin, it seems like whenever a non-CSS legal piece of text is typed into my css files, I get Atom's big orange warning boxes in the upper right hand corner screen. For example, if the file had some git conflict markers, such as:

```css
<<<<<<<<
.foo {}
========
.bar {}
>>>>>>>>
```

While typing and trying to resolve this conflict, I will get many many many many bright orange alerts saying:

*CSS Syntax Error*
Unknown word on line 3

Can we turn this off? It's quite annoying.

**Comment by Arcanemagus** [comment]
This could be changed to just a regular message highlighting that entire line, the original intent of the alert was that if the file isn't valid CSS then most of the checks `stylelint` does are disabled till it is fixed.

@1000ch you want me to do a PR for this or do you want to keep it as an alert?

**Comment by 1000ch** [comment]
It's noisy, certainly. CSS error itself should not be reported?

**Comment by Arcanemagus** [comment]
Oh it should definitely be reported, but instead of an Atom alert let's just do a regular linter error message?

**Comment by 1000ch** [comment]
Yeah :+1: 

>a regular linter error message?

I'll send a pull request later.

**Comment by 1000ch** [comment]
@onlywei Try `v1.7.0`?

**Comment by onlywei** [comment]
@1000ch thanks for the quick turnaround! I'll play around with it today and see how it goes.
